packages/components/scripts/post-build/angular.js

```diff
@@ -185,6 +185,15 @@ module.exports = (tmp) => {
 			}
 		];
 
+		if (component.config?.angular?.initValues) {
+			component.config?.angular?.initValues.forEach((init) => {
+				replacements.push({
+					from: `["${init.key}"];`,
+					to: `["${init.key}"] = ${init.value === '' ? '""' : init.value};`
```

Comment:
```suggestion
					to: `["${init.key}"] = ${init.value || '""'};`
```

wouldn't this mean the same, but be simpler ?

Reply:
No, it wouldn't. In your suggestion `init.value` is undefined, but it should be an empty string, because you can use `undefined` as value